### ISSUE TITLE
tabEndNewRow is replacing other rows data

### ISSUE THREAD

**Issue by petermatik** [olifolkerd/tabulator#2825]
Following value:
```
tabEndNewRow: {name:"steve", age:62}, //create new row with this data
```
is replacing data for other rows, however:
```
tabEndNewRow: function(row){
        //row - row component for current last row in table
        return {name:"bob", age:34}; // return data object for new row
    }
```
seems to be working ok

**Comment by olifolkerd** [comment]
Hey @petermatik 

Sorry to hear you are having trouble.

Thanks for getting in touch, Could i ask that if you are reporting a bug that you use the [Bug Issue Template] (https://github.com/olifolkerd/tabulator/issues/new?assignees=&labels=&template=bug_report.md&title=) as it prompts you for all the information we need to help you out.

Cheers

Oli :)

**Comment by olifolkerd** [comment]
Hey @petermatik 

There was an issue with Tabulator not isolating the row object correctly. i have pushed a fix for this to the 4.7 branch that will be released later this month.

Cheers

Oli :)

**Comment by petermatik** [comment]
Thanks!
